primitives/src/asset.rs

```diff
@@ -48,13 +48,14 @@ use serde::{Deserialize, Serialize};
 pub enum Asset<MarketId> {
     CategoricalOutcome(MarketId, CategoryIndex),
     ScalarOutcome(MarketId, ScalarPosition),
-    CombinatorialOutcome,
+    CombinatorialToken(CombinatorialId),
     PoolShare(PoolId),
     #[default]
     Ztg,
     ForeignAsset(u32),
     ParimutuelShare(MarketId, CategoryIndex),
 }
+// TODO Needs storage migration
```

Comment:
Does not need a storage migration, because the enum variant `CombinatorialOutcome` has never been used before in the storage.

Each enum variant is encoded separately in the storage. https://docs.substrate.io/reference/scale-codec/

The two potential storage items that could contain a `CombinatorialOutcome` are [`Markets`](https://github.com/zeitgeistpm/zeitgeist/blob/ca88b16e20f7dca22d41f5ba31648ab6d45f0d8e/zrml/market-commons/src/lib.rs#L251-L252) that contains the `base_asset` as a potential `Asset`. The other is [`Pools`](https://github.com/zeitgeistpm/zeitgeist/blob/ca88b16e20f7dca22d41f5ba31648ab6d45f0d8e/zrml/neo-swaps/src/lib.rs#L150-L151). But the `Asset` `CombinatorialOutcome` was never assigned.